instance/models/mixins/openedx_database.py

```diff
@@ -475,6 +489,56 @@ def _get_rabbitmq_settings(self):
             "EDXAPP_CELERY_BROKER_USE_SSL": True
         }
 
+    def _get_redis_settings(self):
+        """
+        Return dictionary of Redis settings
+        """
+
+        redis_hostname = "{}:{}".format(
+            self.redis_server.instance_host,
+            self.redis_server.instance_port
+        )
+
+        redis_transport_options = {
+            "CELERY_BROKER_TRANSPORT_OPTIONS": {
+                "global_keyprefix": self.redis_key_prefix
+            }
+        }
+
+        # TODO: Validate the key names as I just copy pasted them from RabbitMQ
+        return {
+            "XQUEUE_REDIS_USER": self.redis_username,
+            "XQUEUE_REDIS_PASS": self.redis_password,
+            "XQUEUE_REDIS_VHOST": str(self.redis_server.instance_db),
+            "XQUEUE_REDIS_HOSTNAME": self.redis_server.instance_host,
+            "XQUEUE_REDIS_PORT": self.redis_server.instance_port,
+            "XQUEUE_REDIS_TLS": self.redis_server.use_ssl_connections,
+            "XQUEUE_SESSION_ENGINE": "django.contrib.sessions.backends.cache",
+            "XQUEUE_CACHES": {
+                "default": {
+                    # FIXME: What should be this? Is it needed at all?
```

Comment:
@gabor-boros I have no idea about this sorry. :/

Reply:
@swalladge Well, I believe this is not needed at all. Based on the [changelog](https://github.com/edx/configuration/blob/b2bf69e0dad429792693f4365827e498811b3991/CHANGELOG.md), XQueue configuration was removed on 2017-02-27 and the remaining config got removed on 2018-04-13.

I believe we could remove all XQueue config 😅 For Redis settings, I removed as part of 56aa44b9.

Do you agree with me?

Reply:
Lol ok cool.  I agree. :)